APILambda/env.ts

```diff
@@ -30,4 +31,5 @@ export type ServerEnvironment = {
   environment: EnvironmentType,
   eventStartWindowInHours: number,
   maxArrivals: number,
+  setProfileCreatedAttribute: (userId: string) => PromiseResult<unknown, unknown>
```

Comment:
Separate the concerns so we know which env variables are used for which routes only